genienlp/run_bootleg.py

```diff
@@ -182,6 +187,9 @@ def parse_argv(parser):
     parser.add_argument(
         '--almond_domains', nargs='+', default=[], help='Domains used for almond dataset; e.g. music, books, ...'
     )
+
+    parser.add_argument('--prep_test_too', action='store_true', help='Prepare bootleg features for test set too')
```

Comment:
`--data_splits {train|dev|test}` or similar would be cleaner...

Reply:
sure